### ISSUE TITLE
Add USD value of balance/staked amounts on Wallet page

### ISSUE THREAD

**Issue by adamkdean** [edge/wallet#169]
Using the index `/token/current` [endpoint](https://index.xe.network/token/current), we're able to discern the USD value of the XE in a wallet.

It would be great to add the USD value inline next to the balance and the staked amount in the Wallet overview. Some attention will need to be paid to the design and how we fit that information in though, to make sure it looks good.

**Comment by annybs** [comment]
We now have basic USD display for the wallet balance, where the amount is shown as `123.45USD`

This should be expanded to include the dollar symbol, ergo `$123.45USD`

I think may be best done by enhancing the Amount component with extra computations ala:

```
computed {
  currencySymbolBefore() { /*...*/ },
  currencySymbolAfter() { /*...*/ }
}
```

These would refer to a static configuration where we can specify for instance,

```
currencySymbols = [
  USD: {
    before: '$',
    after: ' USD'
  }
]
```

On symbol match, use as specified; if no match, use symbol as given (such as XE).

**Comment by adamkdean** [comment]
@annybs could you confirm whether this ticket needs to remain open or can be closed?

**Comment by annybs** [comment]
I can confirm USD, XE, and EDGE amounts display correctly at this time. As we don't currently support other currencies and my previous comment is just a suggestion, I'm happy to close this down and consider future changes a separate task.
